infra/terraform/cloudwatch.tf

```diff
@@ -27,6 +27,7 @@ module "cloudwatch_cis-alarms" {
 }
 
 locals {
+  alb_id = "app/k8s-formbricks-21ab9ecd60/342ed65d128ce4cb"
```

Comment:
could this also be a dynamic value because the load balancer is created by terraform?
As I understood it we do it with variables in other places of the terraform script.